README.md

```diff
@@ -162,7 +45,7 @@ This means that new versions are submitted to a given channel.
 
 Current versions per each channel are represented by the [ModuleTemplate CR](https://github.com/kyma-project/lifecycle-manager/blob/main/docs/technical-reference/api/moduleTemplate-cr.md) submitted to a matching folder in the Kyma Git repository.
 
-Having merged all the changes into the `main` branch in the `keda-manager` repository, the CI/CD jobs will bundle module images and generate ModuleTemplate for you.
+Having merged all the changes into the `main` branch in the `keda-manager` repository, the [CI/CD jobs](/docs/CI-CD-description.md) will bundle module images and generate ModuleTemplate for you.
```

Comment:
```suggestion
Having merged all the changes into the `main` branch in the `keda-manager` repository, the [CI/CD jobs](/docs/CI-CD-description.md) bundles module images and generates a ModuleTemplate for you.
```

or 

```suggestion
Having merged all the changes into the `main` branch in the `keda-manager` repository, the [CI/CD jobs](/docs/CI-CD-description.md) bundles module images and generates the ModuleTemplate for you.
```


"Having merged" is such an impersonal phrase. Who or what does the merging? Can we use active voice with a subject in this sentence?